src/create-matcher.js

```diff
@@ -32,7 +32,7 @@ export function createMatcher (
     createRouteMap([route || parentOrRoute], pathList, pathMap, nameMap, parent)
 
     // add aliases of parent
-    if (parent) {
+    if (parent && parent.alias) {
```

Comment:
So you meant

```suggestion
    if (parent && parent.alias.length) {
```

? Otherwise it would always execute anyway

Reply:
oh~ yes,  you are right! Thanks for your suggestion.

Reply:
oh~ yes,  you are right! Thanks for your suggestion.